admin/partials/admin-monk-select-render.php

```diff
@@ -10,19 +10,19 @@
  * @subpackage Monk/Admin/Partials
  */
 
-	$options = get_option( 'monk_language' );
+	$options = get_option( 'monk_default_language' );
 ?>
-	<select name="monk_language[language_select]">
-		<option value="pt-br"<?php selected( $options['language_select'], 'pt-br' ); ?>>
+	<select name="monk_default_language">
```

Comment:
Create an array called `$available_languages` and do a loop to render the language options.